client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -97,14 +94,13 @@ class AuthorMappingPane extends PureComponent {
 			return this.props.translate(
 				'There are multiple authors on your %(sourceType)s site. ' +
```

Comment:
Maybe use "original" here for specificity.
```suggestion
				'There are multiple authors on your original %(sourceType)s site. ' +
```